### PR TITLE
New Version: Typst.Typst version 0.6.0

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
Finishing Updating the Typst.Typst application. 

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111477&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/t/Typst/Typst/0.6.0/Typst.Typst.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Typst.Typst
+PackageVersion: 0.6.0
+InstallerLocale: en-US
+Installers:
```
Comment (stephengillie) [inline manifests/t/Typst/Typst/0.6.0/Typst.Typst.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by BrandonWanHuanSheng** [comment]
Don't know why the pipelines don't want to work.

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie Why the Azure Pipelines quotes like that? It also my pull request did not have label.


> No commit pushedDate could be found for PR 111477 in repo microsoft/winget-pkgs

**Comment by BrandonWanHuanSheng** [comment]
Attempting for another retry.

**Pull request closed without merging**